python/paddle/optimizer/lr.py

```diff
@@ -55,6 +55,26 @@ class LRScheduler:
 
     LRScheduler Base class. Define the common interface of a learning rate scheduler.
 
+    There are currently 14 strategies implemented in paddle based on this base class, which are:
+
+    - ``NoamDecay``: Related algorithms are derived from `*Attention Is All You Need* <http://blog.inkypy.com>`_ . Please refer to :ref:`en_api_paddle_optimizer_lr_NoamDecay`.
```

Comment:
在英文文档中，英文 api 文档的 label 是由路径转化来的，如该例中的 `paddle.optimizer.lr.NoamDecay`，label应该是`api_paddle_optimizer_lr_NoamDecay`，中文的label则会在前面加 `cn_`。
```suggestion
    - ``NoamDecay``: Related algorithms are derived from `*Attention Is All You Need* <http://blog.inkypy.com>`_ . Please refer to :ref:`api_paddle_optimizer_lr_NoamDecay`.
```
下面每一条都按上述修改～

Reply:
已修改~